Makefile

```diff
@@ -3,6 +3,14 @@ ifeq ($(CONTAINER_ENGINE),)
 	CONTAINER_ENGINE=$(shell podman version >/dev/null 2>&1 && echo podman)
 endif
 
+# see https://github.com/containerd/nerdctl/blob/main/docs/multi-platform.md
+# e.g use CONTAINER_PLATFORM=amd64 for building x86_64 on arm.
```

Comment:
```suggestion
# e.g use CONTAINER_PLATFORM_FLAG=amd64 for building x86_64 on arm.
```

Reply:
no, notice that I had to change it because it failed otherwise. The user needs to set the CONTAINER_PLATFORM=amd64 and then the Makefile will sets the CONTAINER_PLATFORM_FLAG=--platform=amd64 (approppriately) for all the container build calls...